advcpmv-0.8-8.32.patch

```diff
@@ -428,58 +538,63 @@ diff -aur coreutils-8.32/src/cp.c coreutils-8.32-patched/src/cp.c
 +        int j;
 +
 +        /* how many files are we copying */
-+        char command[1024];
-+        sprintf( command, "find \"%s\" -type f | wc -l", file[0]);
 +        FILE *fp ;
 +        char output[1024];
-+        fp = popen(command,"r");
-+        if ( fp == NULL || fgets(output, sizeof(output)-1, fp) == NULL)
-+            printf("failed to run find.\n");
++        char fcmd[] = "find";
++        fp = spawn(fcmd, (char *[]){ fcmd, file[0], NULL, "-type", "f", NULL });
```

Comment:
NULL is used as a terminator, same way argc/argv works. Shouldn't be used in the middle.

```suggestion
+        fp = spawn(fcmd, (char *[]){ fcmd, file[0], "-type", "f", NULL });
```

Reply:
Will do.